server/src/main/java/org/opensearch/cluster/service/ClusterService.java

```diff
@@ -183,6 +183,10 @@ public ClusterState state() {
         return clusterApplierService.state();
     }
 
+    public ClusterState commitState() {
```

Comment:
we dont need this method, you can access LastAcceptedClusterState via Coordinator.getLastAcceptedState()

Reply:
Wouldn't this be returning state `Coordinator.getLastAcceptedState` that is not yet comitted by cluster-manager ? Looks like lastAcceptedState is being updated as part of publish as well before it is committed.